ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoUtil.java

```diff
@@ -53,14 +50,15 @@ public static IteratingLinearOptimizer createLinearOptimizer(RaoParameters raoPa
         if (raoParameters.getObjectiveFunction().equals(RaoParameters.ObjectiveFunction.MAX_MIN_MARGIN_IN_AMPERE)
             || raoParameters.getObjectiveFunction().equals(RaoParameters.ObjectiveFunction.MAX_MIN_MARGIN_IN_MEGAWATT)) {
             fillers.add(new MaxMinMarginFiller(raoParameters.getObjectiveFunction().getUnit(), raoParameters.getPstPenaltyCost()));
+            fillers.add(new MnecFiller(raoParameters.getMnecAcceptableMarginDiminution(), raoParameters.getMnecViolationCost(), raoParameters.getMnecConstraintAdjustmentCoefficient()));
```

Comment:
FYI this is this part of the code that I think we must refactor in another pull request. Here I would put RaoParameters as the unique argument of the MnecFiller constructor. However this is not possible because RaoParameters cannot be imported in rao_commons package otherwise it would cause a cyclic dependency. That is why I think we must move RaoParameters and some methods of RaoUtil into the rao_commons package. This must be discussed further and can wait a little bit.

Reply:
An advantage of the current implementations remains that the fillers only access the parameters they need in RaoParameters. This prevents them from doing things that are not in their scope.
Of course we will discuss this further